content/articles/faq-ssl-certificates.markdown

```diff
@@ -0,0 +1,79 @@
+
+1.  #### I want to purchase an SSL certificate. Do I need a subscription?
+
+    Yes. As explained in our previous FAQ, the SSL certificates are one of our paid products, provided s part of our subscription services.
+
+    In order to purchase an SSL certificate you need an account active subscription at the time of the purchase. You are not required to have an active subscription to use or install the certificate on your domain/server, however please note that if you disable the subscription you will not receive any additional service for the domain or the certificate.
+
+    You will also need a valid, active subscription at the time you will need to renew the certificate, or if you want to reissue the certificate.
+</div>
+
+## Domains and SSL certificates
+
+<div class="section-faq" markdown="1">
+1.  #### Do I need to transfer the domain to you to purchase an SSL certificate?
+
+    No. You can purchase an SSL certificate for a domain that is not registered with us, without transferring it to us. However, you will need to add the domain to your account in order to be able to request the certificate. You also need a valid, active subscription.
+
+    Just follow the general instruction explained in the [Getting Started](/articles/getting-started-ssl-certificates/) guide. When you are asked to add the domain to purchase the certificate, select the option to add a domain without transferring or registering it.
+
+1.  #### Do I need to host the domain with you to purchase an SSL certificate?
+
+    No. You can purchase an SSL certificate for a domain that is not hosted with us. However, you will need to add the domain to your account in order to be able to request the certificate. You also need a valid, active subscription.
+
+    Just follow the general instruction explained in the [Getting Started](/articles/getting-started-ssl-certificates/) guide. When you are asked to add the domain to purchase the certificate, select the option to add a domain without transferring or registering it.
+</div>
+
+
+## Types of Certificates
+
+<div class="section-faq" markdown="1">
+1.  #### What type of SSL certificates does DNSimple provide?
+
+    We currently offer different types of _domain-validated_ SSL certificates, signed by globally recognized certificate authorities. We don't provide _organization-validated_ (OV) or _extended-validation_ (EV).
+
+    To learn more visit the [SSL Certificates](/articles/ssl-certificates/) article.
+
+1.  #### What certificates show the company name in the green bar near the padlock?
+
+    The green bar in the browser is displayed only if you purchase an _extended-validation_ (EV) certificate. We currently don't sell EV certificates.
+
+    We currently only sell DV certificates, that are less expensive than EV certificates and offer the same cryptographic level of security. If you still need/want to purchase an EV certificate, we provide some recommendation at [this page](/articles/can-ev-ssl-certificates/).
```

Comment:
..DV certificates. These are less expensive...
an EV certificate~~,~~ we provide